ambient_toolbox/apps.py

```diff
@@ -1,7 +1,15 @@
 from django.apps import AppConfig
+from django.conf import settings
+from django.core.checks import Tags, register
 from django.utils.translation import gettext_lazy as _
 
 
 class AmbientToolboxConfig(AppConfig):
     name = "ambient_toolbox"
     verbose_name = _("Ambient Toolbox")
+
+
+if getattr(settings, "STATIC_ROLE_PERMISSIONS_PATH", None):
+    from ambient_toolbox.static_role_permissions.system_check import check_permissions_against_models
+
+    register(check_permissions_against_models, Tags.models)
```

Comment:
question: warum Tags.model?

Reply:
Because the system check is related to models, isn't it?

These are the offered tags.

```py
    admin = "admin"
    async_support = "async_support"
    caches = "caches"
    compatibility = "compatibility"
    database = "database"
    files = "files"
    models = "models"
    security = "security"
    signals = "signals"
    sites = "sites"
    staticfiles = "staticfiles"
    templates = "templates"
    translation = "translation"
    urls = "urls"
```

Reply:
ah, ok 👍